flow/Arena.cpp

```diff
@@ -343,6 +344,22 @@ void* ArenaBlock::allocate(Reference<ArenaBlock>& self, int bytes, IsSecureMem i
 	return result;
 }
 
+force_inline ArenaBlock* newBigArenaBlock(int size) {
```

Comment:
It is a performance critical path. Not sure if it is a good idea to introduce an additional check here just for testing purposes. If would add an ifdef disabling it at least for release builds

Reply:
Fair point, but it essentially adds just one conditional branch (with hints for bias).
We do have quite a few `g_network->isSimulated()` checks in production code already.
Also we don't have the concept of release builds, either.
And to test only on non-release builds would also compromise the strength of testing.

Reply:
Perfsanity numbers in comment.